src/components/bcros/filing/addStaffFiling/modal/Form.vue

```diff
@@ -341,6 +343,26 @@ const handleSubmit = () => {
                   validate()
                 }"
               />
+              <BcrosDocumentUpload
+                v-else
+                v-model="staffNotation.courtOrderFile"
+                class="flex-grow"
+                :max-size="MAX_FILE_SIZE"
+                :page-size="PageSizeE.LETTER_PORTRAIT"
+                :disabled="submissionInProgress"
+                :upload-document-to-drs="uploadDocumentToDRS"
+                name="courtOrderFile"
+                data-cy="court-order-upload"
+                @set-error="(error) => {
+                  fileError = error
+                  validate()
+                }"
+                @clear-selection="() => {
+                  staffNotation.courtOrderFile = undefined
+                  fileError = undefined
+                  validate()
+                }"
+              />
```

Comment:
A very small thing but perhaps swap the hierarchy of these two components so that the condition, like `enableThisFeature..` etc is toggling it's associated component vs enabling the fallback component when the enabling condition is false. 
Sorry, hard to articulate that lol, but yea i'd swap so that the condition is on it's respective component

Reply:
100% agree